openmetadata-ui/src/main/resources/ui/cypress/common/common.js

```diff
@@ -688,8 +692,7 @@ export const softDeleteUser = (username, isAdmin) => {
   verifyResponseStatusCode('@searchUser', 200);
 
   // Click on delete button
-  cy.get(`[data-testid="delete-user-btn-${username}"]`).click();
-
+  cy.get(':nth-child(4) > .ant-space > .ant-space-item > .ant-btn').click();
```

Comment:
We should use data-testid instead of class base selector

Reply:
Done